csharp/BYOAuthFlow/Program.cs

```diff
@@ -47,7 +47,7 @@ internal class Program
         ///     The SpatialOS Platform refresh token of a service account or a user account.
         /// </summary>
         private static string RefreshToken =>
-            Environment.GetEnvironmentVariable("IMPROBABLE_REFRESH_TOKEN") ?? "5b6c50ed-f488-45f2-b02b-e6caa346b26e";
+            Environment.GetEnvironmentVariable("IMPROBABLE_REFRESH_TOKEN") ?? "";
```

Comment:
Should there be an empty refresh token check somewhere when we use this or is it in the SDK?